components/drawer/drawer.service.module.ts

```diff
@@ -5,5 +5,9 @@
 
 import { NgModule } from '@angular/core';
 
-@NgModule()
+import { NzDrawerService } from './drawer.service';
+
+@NgModule({
+  providers: [NzDrawerService]
+})
 export class NzDrawerServiceModule {}
```

Comment:
Is it possible to provide services directly on NzDrawerModule instead of NzDrawerServiceModule?
For NzDrawerServiceModule we can delete it

Reply:
Yeah of course i will do it then merge it ;)

Reply:
> Yeah of course i will do it then merge it ;)

@HyperLife1119 seems good now :)